metron-analytics/metron-profiler-client/src/main/java/org/apache/metron/profiler/client/window/WindowProcessor.java

```diff
@@ -89,7 +89,7 @@ public void exitIdentifier(WindowParser.IdentifierContext ctx) {
     if(checkForException(ctx)) {
       return;
     }
-    stack.push(new Token<>(ctx.getText().substring(1), String.class));
+    stack.push(new Token<>(ctx.getText().substring(1), String.class, null));
```

Comment:
Yeah, the polymorphic constructor was added after the fact.  I can remove these now.  Good catch.